engine/univNames.mli

```diff
@@ -8,15 +8,23 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
+open Names
 open Univ
+open Sorts
 
 (** Local universe name <-> level mapping *)
 
-type universe_binders = Univ.Level.t Names.Id.Map.t
+type universe_binders = QVar.t Id.Map.t * Level.t Id.Map.t
+
+type rev_binders = Id.t QVar.Map.t * Id.t Level.Map.t
 
 val empty_binders : universe_binders
 
+val empty_rev_binders : rev_binders
+
 type univ_name_list = Names.lname list
 
-val pr_with_global_universes : ?binders:universe_binders -> Level.t -> Pp.t
+type full_name_list = lname list * lname list
+
+val pr_level_with_global_universes : ?binders:universe_binders -> Level.t -> Pp.t
```

Comment:
https://github.com/coq/coq/blob/1e7473812cec6e735394ca5f5fbefb9c78600893/kernel/univ.mli#L57-L58
The name change was not reflected there